ckan/logic/action/update.py

```diff
@@ -853,36 +859,44 @@ def _group_or_org_reindex(context, data_dict, is_org=False):
     _get_action('task_status_update')({'session': model.meta.create_local_session(), 'ignore_auth': True}, task)
 
     dataset_ids = [dataset.id for dataset in results]
-    job_title = 'Rebuild Dataset Indecies for %s: %s' % ('Organization' if is_org else 'Group', group.name)
-    queue_name = plugins.toolkit.config.get('ckan.search.rebuild_queue_name', 'search_rebuild')
 
-    job = plugins.toolkit.enqueue_job(fn=search.rebuild, title=job_title, queue=queue_name,
-                                      kwargs={'force': True, 'package_ids': dataset_ids, 'in_background': True, 'group_id': group.id})
-
-    task['value'] =  json.dumps({'job_id': job.id, 'total': len(dataset_ids), 'indexed': 0})
+    task['value'] =  json.dumps({'total': len(dataset_ids), 'indexed': 0})
     task['state'] = 'pending'
     task['last_updated'] = str(datetime.datetime.now(datetime.timezone.utc))
 
     _get_action('task_status_update')({'session': model.meta.create_local_session(), 'ignore_auth': True}, task)
 
+    queue_name = plugins.toolkit.config.get('ckan.search.rebuild_queue_name', 'search_rebuild')
+    job = plugins.toolkit.enqueue_job(fn=search_jobs.reindex, title=_('Rebuild Dataset Indices'), queue=queue_name,
+                                      kwargs={'package_ids': dataset_ids, 'group_id': group.id})
+
     return dictize_job(job)
 
 
 # (canada fork only): background search index rebuilding
 #TODO: upstream contrib!!
-def reindex_organization_datasets(context, data_dict):
-    return _group_or_org_reindex(context, data_dict, is_org=True)
+def reindex_organization_datasets_in_background(context, data_dict):
```

Comment:
The pattern (with one exception) for action names is to use `package` and to end the name with the verb. How about:
```suggestion
def organization_packages_background_reindex(context, data_dict):
```

Reply:
@wardi that is done now